content/en/docs/how-tos/interact-with-running-jobs.md

```diff
@@ -0,0 +1,146 @@
+this project will be torn down and terminated.
+
+## Where are the Logs?
+
+From GitHub, you can access the build logs from the `Details` link next to each job listed in the checkbox at the bottom of the PR description
+page. This gives an overall picture of the test output, but you might want to follow each job and test more closely while it runs. In that case,
+from the CI cluster console accessed above, you have a choice between `Administrator` and `Developer` menu. Choose `Administrator` if it is not
+already chosen. You can access the `pods` running the tests from `Workloads` -> `Pods` and then the `Logs` tab across the top. Pods
+usually have multiple containers, and each container can be accessed from the dropdown menu above the logs terminal.
+This is equivalent to running the following from your local terminal, if you are currently logged into the CI cluster:
+
+```bash
+$ oc project ci-op-xxxxxxx
+$ oc get pods
+$ oc logs -f <pod-name> -c <container-name>
+```
+
+## Access the terminal of the pod Running the tests
+
+From the console, you can access the running pod through the `Terminal` tab across the top of the selected pod.
+(`Workloads` -> `Pods` -> `Terminal`). This will give you a shell from which you can check expected file locations,
+configurations, volume mounts, etc. This is useful when setting up a new test workflow to get everything working.
+This is roughly equivalent to the following from your local terminal, if you are currently logged into the CI cluster:
+
+```bash
+$ oc -n ci-op-xxxxxxx rsh <pod-name> -c <container-name>
+```
+
+## Access the external cluster launched with your changes
+If you are debugging a job in which an external cluster was launched (`e2e-aws`, `e2e-*`), you might find it useful to access the cluster.
+After the installer pod has successfully completed (usually this is ~30 min after the job was triggered), you will find the test pod running.
+From the project accessed above, (`Home`->`Projects`) you can grab the `KUBECONFIG` for the external cluster and copy it to your local system,
+to access the cluster against which the extended test suites are running.
+Access the `Terminal` tab of the running test pod (`Workloads` -> `Pods` -> `Terminal`). Find the `kubeconfig` file in the pod's `/tmp` directory.
+Once copied to your local system, you can proceed to run `oc` commands against the cluster by setting `KUBECONFIG` environment variable
+or passing `--kubeconfig` to `oc`. Again, this is intended for WIP PRs only. The cluster will be terminated whenever the job completes.
+It is usually more productive to launch a cluster using `cluster-bot` through `Slack` and manually run `openshift-tests` suites against that, rather
+than through a PR job's cluster. For how to run `openshift-tests` binary and to find more information about the test suites, see
+[How to run the test suites myself outside of CI.](#how-to-run-the-test-suites-myself-outside-of-ci)
+
+## How do I know which tests will run?
```

Comment:
I'd suggest pointing people to the steps.ci.openshift.org viewer - we built it so that you could look at every command that will run for a job.

Reply:
cool! I don't know what that is.. i'll look it up.. in the docs :)

Reply:
I can't find that, but if you point me to it I'll replace/include that

Reply:
found it, nice!! added